### DISCUSSION TITLE
Repository search isn't working anymore?!

### DISCUSSION THREAD

**Discussion by ppo** [community/community#46439] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The last few times I used the "search in this repository", it no longer produce any results.

For example, [searching for `get_wsgi_application`](https://github.com/django/django/search?q=get_wsgi_application) in the [django repository](https://github.com/django/django/) results in "We couldn’t find any code matching 'get_wsgi_application' in django/django"… which isn't correct as it exists in [django/core/wsgi.py](https://github.com/django/django/blob/main/django/core/wsgi.py)!

I tried a few other keywords and no search produced any results.

**Comment by jge162** [comment]
If you want, as it seems search functionality is limited. Maybe an unintended issue atm.  you can use the "in:description" filter to limit your search results to descriptions and README files. See if that helps in the short term. Like if you wanted to search “wsgi” just do “ wsgi in:description “.
